hub/powertoys/grouppolicy.md

```diff
@@ -1,43 +1,128 @@
 ---
 title: PowerToys Group Policy
 description: Group policy objects 
```

Comment:
Wrong description. Please change it to `Group policy documentation` or something similar.